### ISSUE TITLE
Noverlap on sub-set of nodes only.

### ISSUE THREAD

**Issue by tsouk** [jacomyal/sigma.js#721]
Hi @apitts, I'm experimenting with your novelap plugin, and wondered if it was possible to apply it to sub-sets of nodes of a graph only so nothing in the same set is overlapping, but no interactions or calculations happen between sets.

Cheers!

**Comment by apitts** [comment]
@tsouk Yes, I haven't tested it myself but it should be possible to specify a subset of nodes in options.nodes as per this line:
https://github.com/jacomyal/sigma.js/blob/master/plugins/sigma.layout.noverlap/sigma.layout.noverlap.js#L55

**Comment by tsouk** [comment]
excellent!
I'll give it a go. Thank you @apitts !
